apcd-cms/src/apps/view_users/urls.py

```diff
@@ -1,7 +1,8 @@
-from django.urls import path
+from django.urls import path, re_path
 from apps.view_users.views import ViewUsersTable
 
 app_name = 'administration'
 urlpatterns = [
     path('view-users/', ViewUsersTable.as_view()),
+    re_path(r'view-users/?filter=(?P<filter>[\w\ ]+)/', ViewUsersTable.as_view()),
```

Comment:
```suggestion
    path('view-users/<str:filter>/', ViewUsersTable.as_view())
```

Just a suggestion - feel free to ignore. Would there be any instance where the filter will not be passed as a string? If not, you could use this pattern to be a bit more easier to read.

Reply:
I tested this locally and it has the same functionality as the original PR

Reply:
Yeah, this makes sense! Committing.